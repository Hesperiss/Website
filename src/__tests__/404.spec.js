import React from "react";
import { create } from "react-test-renderer";
import Error from '../Components/Shared/404/NotFound';

describe("Page 404", () => {
  test("Matches the snapshot", () => {
    const error = create(<Error/>);
    expect(error.toJSON()).toMatchSnapshot();
  });
});