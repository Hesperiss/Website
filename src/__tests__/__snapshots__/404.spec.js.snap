// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page 404 Matches the snapshot 1`] = `<Error />`;
